README.md

```diff
@@ -6,6 +6,10 @@ Automates installation of the Swift toolchain for Windows hosts on GitHub Action
 ## Usage
 
 > **NOTE:** Only Swift 5.4.2+ is supported
+> 
+> **NOTE:** This action requires a prior
```

Comment:
Is this entirely accurate? Is this needed if you are using lld? If you are using SPM, then the default is lld, and this wouldn't be needed right?

Reply:
I don't know what SPM does, or does by default.  You may be correct about that.  It would be better of course if this were made unnecessary and since the action uses setup-vsdevenv anyway simply did it in the right order, but I was unable to figure out how to make that work.  How would you suggest updating this note?

Reply:
I think that expanding it to be explicit about when it requires that or loosening the assertion that the action "requires" setup-vsdevenv to something along the lines of "may be needed if your build requires finding tools from the MSVC toolset" would be sufficient.